src/UncontrolledAlert.js

```diff
@@ -1,12 +1,22 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Alert from './Alert';
 
-export default class UncontrolledAlert extends Component {
+const propTypes = {
+  fade: PropTypes.bool
+};
+
+const defaultProps = {
+  fade: true
+};
+
+class UncontrolledAlert extends Component {
   constructor(props) {
     super(props);
 
     this.state = { isOpen: true };
     this.toggle = this.toggle.bind(this);
+    this.fade = this.props.fade;
```

Comment:
I don't think any changes are needed to this file, but at the very least, this change does nothing.